.gitignore

```diff
@@ -0,0 +1,3 @@
+/template
```

Comment:
Add `/` to the end to ensure only the exact folder name is ignored. Same for the two below.